obs2ioda-v2/src/cxx/netcdf_variable.h

```diff
@@ -10,7 +10,7 @@ namespace Obs2Ioda {
      *
      * @param netcdfID The identifier of the NetCDF file where the variable will be added.
      * @param groupName The name of the group in which the variable should be created.
-     *                  If nullptr, the variable is added to the root group.
+     *                  If NULL, the variable is added as a global variable.
```

Comment:
In my opinion, these changes should go in a separate PR as they have nothing to do with attributes.